pcdsdevices/beam_stats.py

```diff
@@ -23,6 +32,7 @@ def __init__(self, prefix='', name='beam_stats', **kwargs):
 
 
 class SxrGmd(Device):
+    # TODO: this PV => not found
```

Comment:
Ah! This device can probably be removed. SXR has been disassembled and the GMD was moved into the EBD. Its MJ PVs  are surely completely different now.

Reply:
I removed it in ae61a24